src/main/java/com/mathworks/ci/MatlabCommandStepExecution.java

```diff
@@ -91,12 +90,13 @@ private void createMatlabScriptByName(FilePath uniqeTmpFolderPath, String unique
         // Create a new command runner script in the temp folder.
         final FilePath matlabCommandFile =
                 new FilePath(uniqeTmpFolderPath, uniqueScriptName + ".m");
+        final String cmd = getContext().get(EnvVars.class).expand(getCommand());
```

Comment:
With this change, is the class-level field `env` used anymore? If not, can it be removed along with `setEnv`?

Reply:
Done.